scripts/mint.ts

```diff
@@ -109,6 +109,50 @@ async function mintFor(tokenId: string, recipient: string) {
   );
 }
 
+async function mintForRange(tokenIdRange: string, recipient: string) {
+  const starsRecipient = toStars(recipient);
+
+  // Parse string from "1,10" -> "1" and "10"
+  let split = tokenIdRange.split(",");
+  let start = parseInt(split[0]), end = parseInt(split[1]);
```

Comment:
Maybe this could be simplified by using deconstruction and map? `const [start, end] = tokenIdRange.split(',').map(Number);`

Reply:
Neat trick! I didn't think of this.